src/Entity/Article.php

```diff
@@ -11,11 +12,22 @@ class Article extends News
 {
     const NEWS_TYPE = 'article';
 
+    const CATEGORY_ARTICLE="статья";
+    const CATEGORY_OVERVIEW="обзор";
+    const CATEGORY_ANNOUNCEMENT="объявление";
+
     /**
      * @ORM\ManyToOne(targetEntity="App\Entity\Author", inversedBy="articles")
+     * @Assert\NotBlank()
      */
     private $author;
 
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     * @Assert\NotBlank()
+     */
```

Comment:
@daria-sieroshtan 
You also need [Assert/Choice](https://symfony.com/doc/current/reference/constraints/Choice.html)